network/ovs_networkclient_linux_test.go

```diff
@@ -28,27 +28,6 @@ func TestAddRoutes(t *testing.T) {
 	}
 }
 
-func TestCreateBridge(t *testing.T) {
```

Comment:
This change should be moved into a separate PR. Would like to backport this PR for additional windows coverage on other `release/*` branches.